documentation/develop/develop-application/README.md

```diff
@@ -2,45 +2,53 @@
 
 ## Introduction
 
-You will now develop the application with SAP Business Application Studio visual tools.
+In this section you will first create your development environment in SAP Business Application Studio. Then you will create data models to hold the data, define relationships between the entities and upload some first test data. In addition, you will build a user interface to display this data. As last step you will add a role concept to define different authorization. 
```

Comment:
Suggestion:
Shall we make this as bullet points as there are multiple things we do.
Something like ..

In this section you will first create your development environment in SAP Business Application Studio and then do the following,

- Create Data models 
- Define relationships between the entities
- Upload test data
- Build User Interface to display the data
- Add Role concept to define different authorizations